packages/bundler/src/Bundler.ts

```diff
@@ -156,14 +156,18 @@ export class Bundler implements IBundler {
                   if (confirmations >= confirmedBlocks) {
                     clearInterval(intervalId);
                     resolve(userOpResponse);
+                    return;
```

Comment:
Omitting the return statement in this case is common and doesn't affect the functionality of the code as it doesn't make practical difference.

Reply:
True, in the current state it does not make any practical difference.
I added it to skip the timeout check below which is not needed, to save a few CPU cycles.

The [`Promise` docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/Promise) cite that calling `reject` after `resolve` does not have any effect, and [`clearInterval`](https://developer.mozilla.org/en-US/docs/Web/API/clearInterval) is idempotent, so this `return` statement can safely be removed.
But, in case side-effects are added below, and for performance's sake, it is cleaner IMHO to keep this `return` statement.

If it's still your opinion that I should remove it though, I will do so.

Reply:
I see!